Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -49,6 +52,8 @@ public override void Start()
 
                 if (!string.IsNullOrEmpty(minto))
                 {
+                    this.logger.LogInformation($"Mining enabled.");
```

Comment:
i quite dislike using interpolated strings where it is actually not used at all inside that particular string

also, do not use interpolated string in logging

Reply:
Ah but this is not perf critical (but ok)